generate/generate.go

```diff
@@ -21,7 +21,14 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec       *rspec.Spec
+	ExportOpts *ExportOptions
```

Comment:
if, of course, it isn't I'm all for moving that to `Save*` but afaict it's a generator option